py_gen/templates/_ofclass.py

```diff
@@ -2,9 +2,18 @@
 :: from loxi_ir import *
 :: import py_gen.oftype
 :: import py_gen.util as util
-:: type_members = [m for m in ofclass.members if type(m) == OFTypeMember]
```

Comment:
(q about list comprehensions also applies in other places, e.g. here)

Reply:
please see above for list comprehension unrolling as a workaround...